src/app/app-routing.module.ts

```diff
@@ -4,30 +4,52 @@ import {Routes, RouterModule} from '@angular/router';
 import {CoursesPageComponent} from './modules/courses/courses-list/pages/courses-page.component';
```

Comment:
Use import module for true lazy loading
https://angular.io/guide/router#lazy-loading-route-configuration